test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,33 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
```

Comment:
In the future, make clean PRs!
Since these changes are from last week's LO (and are unrelated to this PR), they should not appear here.